x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/integration/MonitoringIT.java

```diff
@@ -127,7 +126,7 @@ public void testMonitoringBulk() throws Exception {
 
             final MonitoringBulkResponse bulkResponse =
                     new MonitoringBulkRequestBuilder(client())
-                            .add(system, null, new BytesArray(createBulkEntity().getBytes("UTF-8")), XContentType.JSON,
+                            .add(system, "monitoring_data_type", new BytesArray(createBulkEntity().getBytes("UTF-8")), XContentType.JSON,
```

Comment:
Why add in a default type here ?

Reply:
@pickypg can correct me if I'm wrong but I believe this argument is the monitoring data type, e.g. `beats_stats` or `logstash_state`, etc. that are sent in to the `POST _xpack/monitoring/_bulk` API. So I wanted to be more explicit about what the type is, so we don't confuse it in the future with `_type`.

Reply:
They are. We use that as a mechanism for routing the document to the right index.

Reply:
thanks, i see now.